include/libopencm3/stm32/f4/rcc.h

```diff
@@ -699,8 +708,8 @@
 #define RCC_PLLSAICFGR_PLLSAIN_SHIFT		6
 #define RCC_PLLSAICFGR_PLLSAIN_MASK		0x1FF
 
-
 /* --- RCC_DCKCFGR values -------------------------------------------------- */
+
```

Comment:
whitespace changes.

Reply:
also meld.. i tried to make f4 and f7 as equal as possible to spot real differences